zero/src/bin/rpc.rs

```diff
@@ -206,6 +206,9 @@ async fn main() -> anyhow::Result<()> {
     tracing_subscriber::Registry::default()
         .with(
             tracing_subscriber::fmt::layer()
+                // With the default configuration trace information is written
+                // to stdout, but we already use stdout to write our payload (the witness).
+                .with_writer(std::io::stderr)
```

Comment:
is it still relevant? Are we keeping logs related to JDT fetching post debugging?

Reply:
I would say it's relevant as long as we initialize the tracing subscriber.

Reply:
(there is never a time where we want tracing information written to `stdout`)